packages/cxl-lumo-styles/scss/themes/vaadin-accordion.scss

```diff
@@ -3,3 +3,8 @@
   column-rule: 1px solid var(--lumo-shade-10pct); // match panel `outline`
   column-width: calc(var(--cxl-wrap-width) / 4);
 }
+
+
+:host([theme~="cxl-hub-sidebar"]) {
+  // background: pink;
+}
```

Comment:
Commented out code left in?

Reply:
right, leftover from the previous branch, removed